airbyte-integrations/connectors/source-qualaroo/source_qualaroo/schemas/responses.json

```diff
@@ -2,66 +2,66 @@
   "type": "object",
   "properties": {
     "id": {
-      "type": "integer"
+      "type": ["null", "integer"]
     },
     "time": {
-      "type": "string",
+       "type": ["null", "string"],
       "format": "date-time"
     },
     "identity": {
       "type": ["null", "string"]
     },
     "page": {
-      "type": "string"
+       "type": ["null", "string"]
     },
     "referrer": {
       "type": ["null", "string"]
     },
     "user_agent": {
-      "type": "string"
+       "type": ["null", "string"]
     },
     "nudge_id": {
-      "type": "integer"
+      "type": ["null", "integer"]
     },
     "nudge_name": {
-      "type": "string"
+       "type": ["null", "string"]
     },
     "anon_visitor_id": {
-      "type": "string"
+       "type": ["null", "string"]
     },
     "ip_address": {
-      "type": "string"
+       "type": ["null", "string"]
     },
     "answered_questions": {
       "type": "array",
       "items": {
-        "type": "object"
+        "type":["null",  "object"]
       }
     },
     "properties": {
-      "type": "object"
+      "type":["null",  "object"]
     },
     "nps": {
-      "type": "object",
+      "type": ["null", "object"],
```

Comment:
Can I remove this from the `response.json` and still be able to pull it from the `_AIRBYTE_RAW_RESPONSES` table?

Reply:
No it won't add to raw if you remove from schema.